config/config.yaml

```diff
@@ -50,6 +50,7 @@ queryConfig:
     serverAddress: ${ASTRA_QUERY_SERVER_ADDRESS:-localhost}
     requestTimeoutMs: ${ASTRA_QUERY_REQUEST_TIMEOUT_MS:-5000}
   defaultQueryTimeoutMs: ${ASTRA_QUERY_DEFAULT_QUERY_TIMEOUT_MS:-3000}
+  zipkinDefaultMaxSpans: ${ASTRA_QUERY_ZIPKIN_DEFAULT_MAX_SPANS:-20000}
```

Comment:
Also, adding docs for `zipkinDefaultMaxSpans` hereish: https://github.com/airbnb/kaldb/blob/airbnb-main/docs/topics/Config-options.md?plain=1#L315 would be good.